src/components/tab/tab.test.tsx

```diff
@@ -11,11 +11,14 @@ const defaultTab: TabProps = {
   handleClick: clickEvent,
 };
 
-function sut(props: TabProps = defaultTab): HTMLElement {
+function sut(props: TabProps = defaultTab) {
   render(<IonTab {...props} />);
-  return screen.getByRole('button', { name: props.label });
 }
 
+const getTab = () => {
+  return screen.getByTestId('ion-tab');
+};
```

Comment:
```suggestion
const getTab = (id='ion-tab') => {
  return screen.getByTestId(id);
};
```

To reuse in lines 125 and 131